migrations/20190211111550-create-article.js

```diff
@@ -1,30 +1,29 @@
 module.exports = {
   up: (queryInterface, Sequelize) => {
-    return queryInterface.createTable("articles", {
-      id: {
-        allowNull: false,
-        unique: true,
-        primaryKey: true,
-        type: Sequelize.UUID,
-        defaultValue: Sequelize.UUIDV4
+    return queryInterface.createTable(
+      "articles",
+      {
+        id: {
+          allowNull: false,
+          unique: true,
+          primaryKey: true,
+          type: Sequelize.UUID,
+          defaultValue: Sequelize.UUIDV4
+        },
+        title: { type: Sequelize.STRING, allowNull: false },
+        description: { type: Sequelize.TEXT, allowNull: false },
+        body: { type: Sequelize.TEXT, allowNull: false },
+        userId: {
+          allowNull: false,
+          type: Sequelize.UUID,
+          references: { model: "users", key: "id" }
+        },
+        createdAt: { allowNull: false, type: Sequelize.DATE },
+        updatedAt: { allowNull: false, type: Sequelize.DATE },
+        deletedAt: { type: Sequelize.DATE }
       },
-      title: { type: Sequelize.STRING, allowNull: false },
-      description: { type: Sequelize.TEXT, allowNull: false },
-      body: { type: Sequelize.TEXT, allowNull: false },
-      userId: {
-        allowNull: false,
-        type: Sequelize.UUID,
-        references: { model: "users", key: "id" }
-      },
-      createdAt: {
-        allowNull: false,
-        type: Sequelize.DATE
-      },
-      updatedAt: {
-        allowNull: false,
-        type: Sequelize.DATE
-      }
-    });
+      { paranoid: true }
```

Comment:
Please my dear friend @Musigwa , 
anytime you add a column to a table please, **please create a separate migration file which add those columns** ....
I beg you!

Reply:
Feedback is taken, I'm sorry for the inconvenience, I now understand the trouble my way can cause to our codebase especially in data modeling. Next time, I'll make sure to do it in the right way. Thanks.